### PR TITLE
Mobile changes. Connected to edenlabllc/ehealth.api#1241

### PR DESCRIPTION
**Pull request by kiurchv**
* Rewrite map
* Change tabs UI
* Change join rules UI
* Minor UI fixes

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -156,15 +139,15 @@ <h3>Зворотній зв&#8217;язок</h3>
...
+                    <button class="btn feedback__submit" type="submit">
                         надіслати
                     </button>
                     <a class="feedback__mailto" hidden></a>
                 </div>
             </form>
         </article>
     </section>
-  </div>
+  </>
```
Comment (LehaMotovilov) [inline dist/index.html]: Is it valid?

File: src/scripts/react/components/Join.js [hunk 2]
```diff
@@ -186,30 +181,32 @@ const Join = () => (
...
+                    />
+                  </div>
                 </li>
               </ul>
               <h3 className="join-rules__sub-title">ДАЛІ</h3>
               <ul className="join-rules__next-list join-rules__next-list_three">
                 <li className="join-rules__next-list-item">
                   На сайті eHealth в розділі “Мапа” <br />
-                  ознайомитись з
+                  ознайомитись з{" "}
```
Comment (LehaMotovilov) [inline src/scripts/react/components/Join.js]: What is it? O_o

Comment (kiurchv) [reply src/scripts/react/components/Join.js]: Sure

File: dist/index.html (hunk 1 above)
Comment (kiurchv) [reply dist/index.html]: Nope. I'll fix it

File: src/scripts/react/components/Join.js (hunk 2 above)
Comment (kiurchv) [reply src/scripts/react/components/Join.js]: > In cases where you want to preserve the space adjacent to a newline, you can write {'Monkeys: '} or Monkeys:{' '} in your JSX source.
https://reactjs.org/blog/2014/02/20/react-v0.9.html#jsx-whitespace

**Comment by kiurchv** [comment]
@edenlabllc/dev-frontend, waiting for your reviews

File: src/scripts/react/components/Join.js (hunk 2 above)
Comment (ivchenkodima) [reply src/scripts/react/components/Join.js]: &nbsp; ?

Comment (kiurchv) [reply src/scripts/react/components/Join.js]: Two words separated by a non-breaking space will stick together, this is not our desired behavior.

**Comment by ivchenkodima** [comment]
нужно добавить модификатор. Верхние должны быть space-between.

**Comment by ivchenkodima** [comment]
Тут тоже самое. Плохо выглядит

**Comment by ivchenkodima** [comment]
Тут добавь max-Xaxis-value= Math.max(...values

) * 0.6

**Comment by ivchenkodima** [comment]
нету лого(

**Comment by ivchenkodima** [comment]
Что то пошло не так

**Comment by kiurchv** [comment]
@ivchenkodima Fixed

**Pull request merged**